### PR TITLE
Edits of PyFluent-Visualization doc based on doc visual rendering

### PR DESCRIPTION
**Pull request by PipKat**
I cleaned up the doc files based on viewing the rendered doc and comparing to recent edits made for the PyFluent-Parametric doc.

### CODE REVIEW

**Comment by PipKat** [comment]
@dnwillia-work and @ypatel-qa I cleaned up this doc and am awaiting your reviews.

File: doc/source/api/visualization/graphics.rst [hunk 1]
```diff
@@ -6,8 +6,8 @@ Graphics
 .. autopostdoc:: ansys.fluent.visualization.pyvista.pyvista_objects.Graphics   
 
 In the following example, a ``Graphics`` object is instantiated with a Fluent
-session as its context. The ``Graphics`` object is used to create mesh, contour,
-vector, and surface objects. 
+session as its context. The ``Graphics`` object is used to create a mesh,
+contour, vector, and surface object. The contour object is then deleted.
```
Comment (PipKat) [inline doc/source/api/visualization/graphics.rst:10]:
```suggestion
contour, vector, and surface. The contour is then deleted.
```

File: doc/source/api/visualization/plots.rst [hunk 2]
```diff
@@ -6,8 +6,8 @@ Plots
 
 
 In the following example, a ``Plots`` object is instantiated with a Fluent session
-as its context. The ``Plots`` object is used to generate and display an XY plot and
-monitor plots. 
+as its context. The ``Plots`` object is used to generate and plot XY plots and
```
Comment (PipKat) [inline doc/source/api/visualization/plots.rst:9]:
```suggestion
as its context. The ``Plots`` object is used to generate and plot two XY plots and
```

File: src/ansys/fluent/visualization/matplotlib/matplot_objects.py [hunk 3]
```diff
@@ -67,7 +73,16 @@ def _init_module(self, obj, mod):
 
 
 class XYPlot(XYPlotDefn):
-    """XY Plot.
+    """Provides for displaying XY plots.
+
+    Parameters
```
Comment (PipKat) [inline src/ansys/fluent/visualization/matplotlib/matplot_objects.py:78]: These parameters are application to several classes, but they are never defined. I added them, but I need someone to add descriptions.

File: src/ansys/fluent/visualization/matplotlib/matplot_windows_manager.py [hunk 4]
```diff
@@ -280,15 +280,15 @@ def save_graphic(
...
 
         Parameters
         ----------
         window_id : str
-            Window id for which graphic should be saved.
+            Window ID for the graphic to save.
         format : str
-            Graphic format. Supported formats are eps, jpeg, jpg,
-            pdf, pgf, png, ps, raw, rgba, svg, svgz, tif and tiff.
+            Graphic file format. Supported formats are EPS, JPEG, JPG,
```
Comment (PipKat) [inline src/ansys/fluent/visualization/matplotlib/matplot_windows_manager.py:290]: When an extension is not used as part of a specific file name, it is to appear in all capital letters.

**Review by ypatel-qa — APPROVED** [review]

**Comment by PipKat** [comment]
@ajain-work Hey there! I'm looking for another reviewer so that we can get this PR merged soon. Thanks!

**Review by ajain-work — APPROVED** [review]

**Pull request merged**